daml-script/dump/src/main/scala/com/daml/script/dump/Main.scala

```diff
@@ -57,29 +60,68 @@ object Main {
     ()
   }
 
+  private def getACS(
```

Comment:
We should move the logic out of `Main` into some more sensible places in separate PRs.